test/ufuzz.js

```diff
@@ -582,6 +582,14 @@ function _createExpression(recurmax, noComma, stmtDepth, canThrow) {
       case 30:
         return '(' + createObjectLiteral(recurmax, COMMA_OK, stmtDepth, canThrow) + '.' +
             SAFE_KEYS[rng(SAFE_KEYS.length)] + ") ";
+      case 31:
+        var name = VAR_NAMES[rng(VAR_NAMES.length)];
+        return '(' + name + ' && ' + name + '[' +
+            createExpression(recurmax, COMMA_OK, stmtDepth, canThrow) + "]) ";
+      case 32:
+        var name = VAR_NAMES[rng(VAR_NAMES.length)];
```

Comment:
I'm saying we could generate unguarded `(x.prop = expr)` and `(x[expr] = expr)` when `canThrow` is true. But only a smaller percentage of the time, otherwise it'd throw too much.